dmaws/rds.py

```diff
@@ -248,13 +248,24 @@ def clean_database(self):
         )
 
         open_framework_ids = self.get_open_framework_ids()
-        self.log("Delete draft services for open frameworks")
+
+        self.log("Delete draft services ")
+        self.log("  > Delete draft services for open frameworks")
         self.cursor.execute("""
             DELETE FROM draft_services WHERE framework_id IN (
                 SELECT id FROM frameworks WHERE status='open'
             )
             """)
 
+        self.log("  > Delete draft services belonging to dangling suppliers")
+        self.cursor.execute("""
+            DELETE FROM draft_services WHERE supplier_id IN (
```

Comment:
Your suggestion works but I think we should keep what I had before because that particular snippet of SQL shows up a few times in this file.